### PR TITLE
[framework] dont hit test for systemnav style if there is no bottom window padding

### PR DESCRIPTION
**Pull request by jonahwilliams**
This means there is no bottom navigation bar, so the hit test is wasted.  (at least, I think so...)

### CODE REVIEW

File: packages/flutter/lib/src/rendering/view.dart [hunk 1]
```diff
@@ -273,22 +273,27 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
...
-      // Horizontal center of the screen
-      bounds.center.dx,
-      // Vertical center of the system navigation bar. The system navigation bar
-      // height is kept as bottom window padding. The "1" needs to be subtracted
-      // from the bottom because available pixels are in (0..bottom) range.
-      // I.e. for a device with 1920 height, bound.bottom is 1920, but the most
-      // bottom drawn pixel is at 1919 position.
-      bounds.bottom - 1.0 - _window.padding.bottom / 2.0,
-    );
     final SystemUiOverlayStyle? upperOverlayStyle = layer!.find<SystemUiOverlayStyle>(top);
```
Comment (goderbauer) [inline packages/flutter/lib/src/rendering/view.dart:276]: Would the same logic apply to the system status bar? Can this be skipped if there isn't viewPadding at the top?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/rendering/view.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter/lib/src/rendering/view.dart:276]: Good idea, let me check that...

Comment (jonahwilliams) [reply packages/flutter/lib/src/rendering/view.dart:276]: can you hide this on iOS or just Android?

Comment (jonahwilliams) [reply packages/flutter/lib/src/rendering/view.dart:276]: I'll follow up with this one another time

**Comment by jonahwilliams** [comment]
g3 testing failures are due to binding change, I'll wait til that is resolved then rebase

**Comment by jonahwilliams** [comment]
or maybe it is already fixed...

**Comment by jonahwilliams** [comment]
So I have failing google3 tests due to the Bindings change, but I already pulled latest. how do I get frob to use a different base commit - do I need to rebase?

**Comment by goderbauer** [comment]
I don't think there is a way for frob to do that. I think you just have to wait until that change rolled into google3, unfortunately. @renyou would know for sure.

**Pull request closed without merging**